### PR TITLE
HMS-5203: Remove template patch banner

### PR DESCRIPTION
**Pull request by Andrewgdewar**
## Summary

Removes a banner that is no longer needed.
Screenshot 2024-12-16 at 2 39 42 PM

## Testing steps
- Without the PR, there is a banner in templates
- After the PR, no banner.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-5203

File: src/Pages/Repositories/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx [hunk 1]
```diff
@@ -268,7 +268,7 @@ const SnapshotListModal = () => {
                         {!checkedSnapshots.size || !rbac?.repoWrite
                           ? 'Remove selected snapshots'
                           : checkedSnapshots.size == count
-                            ? 'Can\'t remove all snapshots'
+                            ? "Can't remove all snapshots"
```
Comment (xbhouse) [inline src/Pages/Repositories/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx:271]: changes in this file triggered a couple warnings, maybe we should change the contraction to `cannot`? or we could do what you've done in this [PR](https://github.com/content-services/content-sources-frontend/pull/397/files)?

**Comment by xbhouse** [comment]
banner is gone 😄  just one small comment about the build warnings, otherwise looks good!

File: src/Pages/Repositories/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx (hunk 1 above)
Comment (Andrewgdewar) [reply src/Pages/Repositories/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx:271]: hmm this is eslint and prettier fighting each other. I should likely solve this. 🤔

Comment (xbhouse) [reply src/Pages/Repositories/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx]: seeing the warnings even with the single quotes 😕 but i think these can be fixed by changing `can't` to `cannot`

**Review by xbhouse — APPROVED** [review]

**Comment by mayurilahane** [comment]
Tested and cannot see the banner now on template list page 

current view is this 
image

**Pull request merged**